easybuild/easyconfigs/s/squashfs-tools/squashfs-tools-4.6.1-GCCcore-12.3.0.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
```

the Makefile has an `install` rule, so we can use `ConfigureMake` here and skip the configure step:
```suggestion
easyblock = 'ConfigureMake'
```